src/ng/log.js

```diff
@@ -11,6 +11,9 @@
  *
  * The main purpose of this service is to simplify debugging and troubleshooting.
  *
+ * To reveal the location of the calls to `$log` in the JavaScript console,
+ * "blackbox" the Angular source in your browser.
```

Comment:
That is a good idea.  I've looked and I can't find a neutral source of sufficient standing to cite.  The closest are the documentation pages from Chrome and Firefox which outline how to set blackboxing up.  I guess we can just drop this PR if one of these links wouldn't suffice:

https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Black_box_a_source
https://developer.chrome.com/devtools/docs/blackboxing